python/tach/cli.py

```diff
@@ -99,19 +99,28 @@ def print_errors(error_list: list[BoundaryError]) -> None:
             build_error_message(error),
             file=sys.stderr,
         )
+    print(
+        f"{BCOLORS.WARNING}\nIf you intended to add a new dependency, run 'tach sync' to update your module configuration."
+        f"\nOtherwise, remove any disallowed imports and consider refactoring.\n{BCOLORS.ENDC}"
+    )
 
 
 def print_unused_dependencies(
     all_unused_dependencies: list[UnusedDependencies],
 ) -> None:
     constraint_messages = "\n".join(
-        f"\t{BCOLORS.WARNING}{unused_dependencies.path} does not depend on: {unused_dependencies.dependencies}{BCOLORS.ENDC}"
+        f"\t{BCOLORS.WARNING}'{unused_dependencies.path}' does not depend on: {unused_dependencies.dependencies}{BCOLORS.ENDC}"
         for unused_dependencies in all_unused_dependencies
     )
     print(
         f"❌ {BCOLORS.FAIL}Found unused dependencies: {BCOLORS.ENDC}\n"
         + constraint_messages
     )
+    print(
+        f"{BCOLORS.WARNING}\nRemove the unused dependencies from tach.yml, "
+        f"or consider running 'tach sync --prune' to update module configuration and "
```

Comment:
Definitely worth considering making --prune the default behavior, obv would make sense as a sep. pr!